assets-src/js/main.js

```diff
@@ -9,14 +11,25 @@ import {flashes} from "./main/flashes";
 
 function domLoadedActions() {
 	accountMenu();
-	navigation();
 	cardEnhancement();
 	collapsibles();
 	disclosureWidget();
 	formErrorSummary();
 	responsiveTables();
 	flashes();
-};
+
+	/* Create a navDoubleLevel object and initiate double-level navigation for a <ul> with the correct data-component attribute */
+	const navDoubleIntro = document.querySelector('ul[data-component="nav-double-intro"]');
+
+	if (exists(navDoubleIntro)) {
```

Comment:
I don't get why we need a dedicated function to check if the element exists.
Isn't checking if `navDoubleIntro` is null enough?

Same comment applies each time that function is used in the other files.

Reply:
Thanks. I've double checked and we can indeed simplify the if statements and remove the that function.

I've just [pushed another commit](https://github.com/w3c/w3c-website-templates-bundle/pull/103/commits/0f645f32f031894cde1a8d45b4a773ecfbc4b39b) for this (and some other bits of tidying in the JS files):